deeplearning4j/src/main/java/com/baeldung/deeplearning4j/cnn/CnnExample.java

```diff
@@ -0,0 +1,21 @@
+package com.baeldung.deeplearning4j.cnn;
+
+
+import com.baeldung.deeplearning4j.cnn.domain.network.CnnModel;
+import com.baeldung.deeplearning4j.cnn.domain.network.CnnModelProperties;
+import com.baeldung.deeplearning4j.cnn.service.dataset.CifarDataSetService;
+import lombok.extern.slf4j.Slf4j;
+import org.deeplearning4j.eval.Evaluation;
+
+@Slf4j
+public class CnnExample {
```

Comment:
Could we move all the classes to a single package and make them all package protected?